### PR TITLE
Document the latest import features for v2.2

### PR DESCRIPTION
**Pull request by sebastienduverne**
## Purpose of the PR
- [x] ABC-189 - Document the curve importer
  - Landing page (index)
    - Add “curves” in “Importing” feature high-level description.
  - Window reference > Alembic Import Settings window
    - Update all screenshots (new settings + page uniformity)
    - Add 2 lines in “Scene options” section table
  - Working with Curves (new page)
    - Provide info about curve data import
    - Provide info about curve rendering

- [x] ABC-198 - Document the Alembic import-less flow
  - Importing Alembic files
    - Better introduce the import topic (minimal)
    - Split the content in 2 sub-pages
    - Basic import page:
      - Contexts of use and requirements
      - Full review for uniformization with the other page
    - External file Import page:
      - Contexts of use and requirements
      - Typical high-level workflow
      - Recommendations
  - Window reference > Alembic Import Settings window
    - Make a clear mention that this window is about importing an Alembic source asset that is located in a folder of the Project window.
    - Review/emphasize the section about Apply/Revert buttons
  - Window reference > Alembic Stream Player component
    - Structure the page to take both import cases into account
    - Update/add screenshots
    - Add all controls for the import-less context
    - Make the content and descriptions that are in common with Alembic Import Settings window reusable (single sourced)

### CODE REVIEW

**Review by clusty — APPROVED** [review]
I am liking it :)

**Review by cguer — APPROVED** [review]
QA approved 👍!

File: com.unity.formats.alembic/Documentation~/import-file-local.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Importing a local Alembic file through your project folder
+
+Use this method when you can have a copy of the Alembic source file in a folder within your Unity project, for example:
+* To import smaller assets.
```
[the remark this answers is not in the document]
Comment (cguer) [reply com.unity.formats.alembic/Documentation~/import-file-local.md:4]: I agree with Vlad, I think it's ok to mention use cases for the "import-less" method but I think we can leave the examples out here. It should be the default workflow unless they have specific needs.

**Review by cguer — APPROVED** [review]
Minor note on use cases for imported workflow but otherwise QA approved 👍!

**Pull request merged**